src/resources/snippets.ts

```diff
@@ -191,4 +191,28 @@ export default [
     "code": ".lyrics-lyrics-background { display: none; } .lyrics-lyrics-contentWrapper>*:not(.lyrics-lyricsContent-active, .lyrics-lyricsContent-highlight, .lyrics-lyricsContent-provider, .lyrics-lyricsContent-description, .lyrics-lyricsContent-unsynced) { color: #FFFFFF4D !important; } .lyrics-lyrics-contentWrapper>*:not(.lyrics-lyricsContent-active, .lyrics-lyricsContent-highlight, .lyrics-lyricsContent-provider, .lyrics-lyricsContent-description, .lyrics-lyricsContent-unsynced):hover { color: #FFFFFF !important; } .lyrics-lyricsContent-highlight { color: #FFFFFF66; } .lyrics-lyricsContent-unsynced { color: #FFFFFF !important; } .lyrics-lyricsContent-unsynced:hover { color: #FFFFFF !important; } .lyrics-lyricsContent-provider, .lyrics-lyricsContent-description { color: #FFFFFFB6 !important; }",
     "preview": "resources/assets/snippets/pretty-lyrics.png",
   },
+  {
+    "title": "Fix artist text glitch",
+    "description": "Fix the problem with new stable relase on linux di spotify",
+    "code": ".eSMjmiD29Ox35O95waw6 { -webkit-mask-image: linear-gradient(90deg,transparent 0,var(--spice-sidebar) 6px,,transparent); }",
+    "preview": "resources/assets/snippets/fix-linux-glitch.png"
+},
+  {
+    "title": "Reduce top padding",
+    "description": "Reduce top padding on new home structure in linux",
+    "code": ".spotify__container--is-desktop .nav-ylx .Root__top-container { padding-top: 10px; }",
+    "preview": "resources/assets/snippets/padding-top.png"
+},
+  {
+    "title": "Make song info bigger",
+    "description": "increse size of song info on play bar and bold song name",
+    "code": ".main-trackInfo-name { font-size: 17px; font-weight: bold; } .main-trackInfo-artists a:link, .main-trackInfo-artists a:visited { font-size: 15px; }",
+    "preview": "resources/assets/snippets/bigger-info.png"
+},
+  {
+    "title": "Playlist on libx make better",
+    "description": "allign icon and text on libx playlist sidebar for better consistency ",
```

Comment:
Is this "linux" or "libx" (the library-x feature I'm assuming)? Does it have images shown in that mode by default, or is it using an extension to add them?
```suggestion
    "description": "Align icon and text on libx playlist sidebar for better consistency ",
```